README.md

```diff
@@ -98,6 +99,45 @@ Xcode is also able to display `TODO` and `FIXME` tags directly in the source nav
 
 Other conventional comment tags, such as `NOTE` are not recognized by Xcode.
 
+### Types ###
+
+Always use Swift native types when available. Swift offers a bridging to Objective-C so you can use them whenever you want.
```

Comment:
I think we can reword this a bit for clarity:

`Prefer Swift native types over Objective-C types when possible. Because Swift types bridge to Objective-C, you should avoid types like NSString and NSNumber in favor of Int or String.`